src/Plugin.NFC/Android/NFC.android.cs

```diff
@@ -24,6 +24,7 @@ public class NFCImplementation : INFC
 		public event TagDiscoveredEventHandler OnTagDiscovered;
 		public event EventHandler OniOSReadingSessionCancelled;
 		public event TagListeningStatusChangedEventHandler OnTagListeningStatusChanged;
+		public event EventHandler OniOSDidInvalidate;
```

Comment:
Does it make sense to have "iOS" in the eventname if we also have to add it to the android class? What would be the disadvantage to have it called "OnDidInvalidate", or lets call it "OnSessionInvalidated"?

Reply:
Since this event does only gets invoked from iOS side I would say lets stay with this naming. Cant see that reusing this event from droid side is possible for now. And having iOS in its name should set awareness that it could only be used on iOS platform implementations. I also tried to stay in line with the naming that the origin creator choosed in the didInvalidate override for OniOSReadingSessionCancelled. see: https://github.com/convivo-gmbh/Plugin.NFC/pull/1/files#diff-48ddbb40d363331f37293a1b58b96e50b954f84895f8e0b7f28b6d31af3b726fR702